clang/lib/CodeGen/CoverageMappingGen.cpp

```diff
@@ -1592,6 +1605,13 @@ struct CounterCoverageMappingBuilder
         llvm::EnableSingleByteCoverage
             ? getRegionCounter(S->getCond())
             : addCounters(ParentCount, BackedgeCount, BC.ContinueCount);
+    auto [ExecCount, ExitCount] =
+        (llvm::EnableSingleByteCoverage
+             ? std::make_pair(getRegionCounter(S), Counter::getZero())
+             : getBranchCounterPair(S, CondCount));
+    if (!llvm::EnableSingleByteCoverage) {
```

Comment:
probably needs to be wrapped in NDEBUG or whatever?

Reply:
I will sweep `llvm::EnableSingleByteCoverage` stuff around here with coming changes. Please be patient.

Reply:
Do you mean that more stuff will be under this `if`?